package.json

```diff
@@ -75,13 +75,13 @@
     "css-loader": "^1.0.1",
     "del": "^3.0.0",
     "devtron": "^1.4.0",
-    "electron": "^3.0.10",
+    "electron": "^4.0.3",
```

Comment:
From what I see, this PR also upgrades Electron and TypeScript versions. Would you mind either:
1. Ideally, splitting that up into a separate PR.
2. Otherwise, update the PR title to reflect this?

Reply:
I will update all versions of them，and create a new PR